src/en/space-station-14/departments/science/proposals/anomalous-research-update.md

```diff
@@ -0,0 +1,31 @@
+ -	If the anomaly is not found within a short period, it will likely cause substantial damage to the station and/or crew, either by going critical or via its pulse effects.
+ -	When the anomaly is located, but it is not in a place that is deemed appropriate or it has an undesirable pulse effect, it is generally removed.
+ -	Once an appropriate anomaly has been found and stabilized so that research on it can commence, it is then generally left to its own devices by the science team until it begins losing stability. Unless the pulse effect is easily countered (e.g., spacing a room to prevent the heat effect of a fire anomaly), the anomaly will continue to damage the station when it pulses, much to the annoyance of station engineers.
+ To summarize the frustrations of this gameplay loop, in the initial stages of anomalous research, the science team is sent on what is essentially a wild goose chase in order to find the anomaly in a timely manner. They will likely need to repeat the search until they achieve what is considered an ‘acceptable’ combination of anomaly type and station location. Even then, conducting/maintaining any anomalous research poses a significant ongoing risk to the crew, with little to benefit them.
+ 
+ ## Proposed changes
+ To alleviate the problems listed above, the following changes to the anomalous research gameplay loop are proposed:
+ -	The science team surveys the station using hand portable tools (i.e., repurposed anomaly detectors) to gather data on potential anomaly sites (i.e., space-time rifts. These rifts are invisible to most players, with some notable exceptions, such as the heretic).
+ -	After surveying the station, the science team returns to their department, downloads their survey data into the anomaly computer (i.e., the former anomaly generator), and consults it to see a map of the station with the rifts that their scanners detected during the survey.
+ -	Rifts have different types and levels of quality - the anomaly they can generate depends on the rift type, and higher-quality rifts will generate anomalies with stronger pulse effects but will generate more research points.
+ -	The science team determines which rift they would like to activate, then heads to its approximate location with a ‘space-time rift stimulation device’ in tow.
+ -	Once at the rift site, the science team secures the area around the rift, potentially with the help of engineering, by building walls and the infrastructure that will be needed to contain the anomaly, as properly contained anomalies will not pulse or destabilize on their own. For example, an electrical anomaly will not pulse if there are a sufficient number of grounding rods nearby, fire anomalies will not pulse if kept sufficiently hot, nor ice anomalies if kept sufficiently cold, etc. Higher-quality anomalies will generally need more infrastructure to remain fully contained.
```

Comment:
this point is strange to me because it technically exists except in a more nuanced and emergent. Way. Ice anomalies don't stop pulsing, but their cooling effect can be counteracted with heaters. Similarly, electricity anomalies can be largely neutralized via the use of grounding rods. You can set up emitters or ship guns to shoot at flesh monsters.

Perhaps it would be better to tie more anomaly effects to physical phenomena, so that there would be more emergent ways of handling them.